index.bs

```diff
@@ -1770,7 +1797,7 @@ session to exist at any particular time, or by providing more complicated sessio
 The following operations can be invoked by the client in an authenticator session.
 
 
-<h4 id="op-make-cred" algorithm>The <dfn>authenticatorMakeCredential</dfn> operation</h4>
+<h4 id="op-make-cred" algorithm>The <dfn export>authenticatorMakeCredential</dfn> operation</h4>
```

Comment:
Other specifications shouldn't be calling this operation, so we shouldn't export it. If it needs to be mentioned non-normatively, the other spec can [use a `spec` attribute to do so](https://tabatkins.github.io/bikeshed/#linking-to-unexported-definitions).